packages/core/src/assets/evaluationTemplates.ts

```diff
@@ -0,0 +1,219 @@
+export const evaluationTemplates = [
+  {
+    title: 'Factuality',
+    template:
+      'Evaluate whether the following response is factually accurate based on the provided context. Return `true` if accurate, `false` otherwise, and provide a reason explaining why. The output should be formatted in JSON.\n\n**Input**:\n\n- Context: {{context}}\n- Response: {{response}}',
+    description:
+      'Assesses the factual accuracy of a response based on given context.',
+    type: 'boolean',
+  },
```

Comment:
Is this how we'll define evaluation templates from now on? No db?

Reply:
most probably yes, but for now i only created those so that i could quickly populate my db of templates in local env